index.js

```diff
@@ -55,9 +55,10 @@ function createAttrString(attrs) {
           return styles[property] != '' ? [camelToDash(property).toLowerCase(), styles[property]].join(':') : '';
         }).filter(removeEmpties).join(';');
       }
-      return styles != '' ? ' style="' + escapeHtml(styles, true) + '"' : '';
+      return styles != '' ? ' style="' + (escapeAttributeValues ? escapeHtml(styles, true) : styles) + '"' : '';
```

Comment:
I think the code would be much simpler if we allow to overwrite the escape-html function. So options with default values might be
```
{
  escapeAttributeValue: escapeHtml,
  escapeString: escapeHtml
}
```
This way the user has much better control over whats happening